Add touch targets guidance to accessibility (#9876)
braze-inc/braze-docs: _docs/_help/accessibility.md

{% endraw %}

### Touch targets

Touch targets are any part of your message that users tap to take action—like buttons, links, or icons. These elements need to be large enough and spaced far enough apart for people to tap them easily, especially on mobile devices.

When touch targets are too small or too close together, it can be frustrating or impossible for users with mobility or dexterity challenges to interact with your message. Improving this can help reduce errors and create a smoother experience for everyone.

Here’s what to keep in mind:

> - **Make it easy to tap.** Aim for a minimum touch target size of 44 x 44 pixels. This aligns with WCAG 2.2 guidelines ([Success Criteria 2.5.8](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)) and common mobile usability standards.

Author: ```suggestion
- **Make it easy to tap.** Aim for a minimum touch target size of 44 x 44 pixels. This aligns with WCAG 2.2 guidelines for [touch targets](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) and common mobile usability standards.
```
```suggestion
- **Make it easy to tap.** Aim for a minimum touch target size of 44 x 44 pixels. This aligns with WCAG 2.2 guidelines ([Success Criteria 2.5.8](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)) and common mobile usability standards.
```

> Touch targets are any part of your message that users tap to take action—like buttons, links, or icons. These elements need to be large enough and spaced far enough apart for people to tap them easily, especially on mobile devices.
> 
> When touch targets are too small or too close together, it can be frustrating or impossible for users with mobility or dexterity challenges to interact with your message. Improving this can help reduce errors and create a smoother experience for everyone.
> 
> Here’s what to keep in mind:
> - **Make it easy to tap.** Aim for a minimum touch target size of 44 x 44 pixels. This aligns with WCAG 2.2 guidelines for [touch targets](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) and common mobile usability standards.
> - **Give each target breathing room.** If tap targets are too close together—like stacked links or tightly grouped buttons—it can be easy to miss or tap the wrong one. Add spacing or padding between elements to prevent that.
> - **Don’t rely on visuals alone.** Even small icons can be made more usable with extra padding, so they meet minimum size requirements without changing the layout.
> - **Preview on mobile.** Test your message on different screen sizes and make sure interactive elements are easy to use.
> 
> Improving touch targets is one of the most effective ways to make your message more accessible on mobile—and it’s good UX for everyone.

Reviewer: ```suggestion
Touch targets are any part of your message that users tap to take action, like buttons, links, or icons. These elements need to be large enough and spaced far enough apart for people to tap them easily, especially on mobile devices.

When touch targets are too small or too close together, it can be frustrating or impossible for users with mobility or dexterity challenges to interact with your message. Improving this can help reduce errors and create a smoother experience for everyone.

Here’s what to keep in mind:
- **Make it easy to tap.** Aim for a minimum touch target size of 44 x 44 pixels. This aligns with WCAG 2.2 guidelines for [touch targets](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) and common mobile usability standards.
- **Give each target breathing room.** If tap targets are too close together—like stacked links or tightly grouped buttons—it can be easy to miss or tap the wrong one. Add spacing or padding between elements to prevent that.
- **Don’t rely on visuals alone.** Even small icons can be made more usable with extra padding, allowing them to meet minimum size requirements without altering the layout.
- **Preview on mobile.** Test your message on different screen sizes and make sure interactive elements are easy to use.

Improving touch targets is one of the most effective ways to make your message more accessible on mobile—and it’s good UX for everyone.
```
